Changelog.md

```diff
@@ -3,6 +3,11 @@ Development version:
 
 * Added the ability to specify automatic alignment of dropped items. (PR #57)
 
+Version 2.0.5 (2016-03-30)
+--------------------------
+
+* Marked strings for translations (PR #66)
```

Comment:
The line above (in "Development Version" needs to be moved into this section since it would be included in 2.0.5.)